iSkyLIMS_drylab/views.py

```diff
@@ -573,18 +641,20 @@ def search_service(request):
                 )
             service_list = []
             for samples_in_service in samples_in_services:
-                service_list.append(samples_in_service.samples_in_service.pk)
+                service_list.append(samples_in_service.samplesInService.pk)
```

Comment:
should be as was before, I mean not camelCase, to match it with the field at models